testing/python/fixture.py

```diff
@@ -490,6 +490,21 @@ def test_func(something): pass
         print(ss.stack)
         assert teardownlist == [1]
 
+    def test_mark_as_fixture_with_prefix_and_decorator_fails(self, testdir):
+        testdir.makeconftest("""
+            import pytest
+
+            @pytest.fixture
+            def pytest_funcarg__marked_with_prefix_and_decorator():
+                pass
+        """)
+        result = testdir.runpytest_subprocess()
+        assert result.ret != 0
+        result.stdout.fnmatch_lines([
+            "*AssertionError: *",
+            "*pytest_funcarg__marked_with_prefix_and_decorator*"
```

Comment:
IIRC the first line will eat the line itself, and the error looks like: `AssertionError: pytest_funcarg__marked_with_prefix_and_decorator`.

Just match that line (as a single line) instead.